src/mlx/warnings.py

```diff
@@ -129,28 +131,47 @@ def check(self, content):
         Args:
             content (str): The content to parse
         '''
-        self.count += len(re.findall(self.pattern, content))
+        matches = re.findall(self.pattern, content, re.M)
+        if self.verbose:
+            for match in matches:
+                print(match)
```

Comment:
this prints the match object and not the original line from the log-file (see run of test cases for details)